app/domain/references/routes.py

```
@@ -154,22 +155,34 @@ async def enhancement_request_hybrid_auth(
 enhancement_request_router = APIRouter(
     prefix="/enhancement-requests",
     tags=["enhancement-requests"],
-    dependencies=[Depends(enhancement_request_hybrid_auth)],
+    dependencies=[
+        Depends(enhancement_request_hybrid_auth),
+        Depends(PayloadAttributeTracer("robot_id")),
```

My gut says this is a little brittle - given the PayloadAttributeTracer fails silently if the attribute is not present in the payload, underlying model changes/field renames etc might cause us to lose tracing for specific attributes in future. Is there a way to add tests to check this?